pkg/handler/server_ssh.go

```diff
@@ -534,6 +514,24 @@ func (s *Server) proxyAssetCommand(sess ssh.Session, sshClient *srvconn.SSHClien
 			}
 		}
 	}
+	wg.Wait()
+	cmd := model.Command{
+		SessionID:   respSession.ID,
+		OrgID:       respSession.OrgID,
+		Input:       rawStr,
+		User:        respSession.User,
+		Server:      respSession.Asset,
+		Account:     respSession.Account,
+		Timestamp:   now.Unix(),
+		DateCreated: now,
+	}
+	outResult := recordBuf.String()
+	cmd.Output = strings.ReplaceAll(outResult, "\x00", "")
+	termCfg := s.GetTerminalConfig()
+	cmdStorage := proxy.NewCommandStorage(s.jmsService, &termCfg)
+	if err2 := cmdStorage.BulkSave([]*model.Command{&cmd}); err2 != nil {
+		logger.Errorf("Create command err: %s", err2)
+	}
 	reason := string(model.ReasonErrConnectDisconnect)
 	s.recordSessionLifecycle(respSession.ID, model.AssetConnectFinished, reason)
 }
```

Comment:
There have been no significant code changes between the two provided snippets, so I can't identify any irregularities, potential issues, or optimizations suggestions from them.

Both functions appear to be written for the `proxyAssetCommand` function and seem to operate in similar ways but with minor adjustments.